src/test/java/net/sf/jabref/importer/fileformat/MsBibImporterTest.java

```diff
@@ -0,0 +1,64 @@
+package net.sf.jabref.importer.fileformat;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.model.entry.BibEntry;
+
+/**
+ *
+ * @author tschechlovdev, mairdl
```

Comment:
The team decided not to put `@author` tags: `git blame` gives the authors even though [git does not directly support multiple authors](http://stackoverflow.com/q/7442112/873282).